src/main/java/net/commments/ft/Test22.java

```diff
@@ -0,0 +1,46 @@
+package net.commments.ft;
+
+import net.commments.object.BCommentPage;
+import net.commments.object.BCommentWindow;
+import net.commments.object.CommentPage;
+import net.commments.object.CommentWindow;
```

Comment:
There is no `net.commments.object.CommentWindow` in the PR. Please add it.

Reply:
done